Processing fails when class member has a comment

# Bug Report

<!-- First, thank you for reporting a bug. That takes time and we appreciate that! -->

| Subject        | Details                                                         |
| :------------- | :---------------------------------------------------------------|
| Rector version | tested on v0.7.25 and dev-master@3383d3d              |
| Installed as   | composer dependency                      |

<!-- Please describe your problem here. -->

```
 ▸ ./vendor/bin/rector process app/Test.php --dry-run
Rector dev-master@3383d3d
Config file: rector.yaml

 3/3 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%

                                                                                                                        
 [ERROR] Could not process "app/Test.php" file by                                                                       
         "Rector\TypeDeclaration\Rector\Property\PropertyTypeDeclarationRector", due to:                                
         "Call to a member function getVarType() on null".
```

## Minimal PHP Code Causing Issue

<!-- Try to reproduce the issue using https://getrector.org/demo/ and post the URL here. -->

```php
<?php

class Test
{
    const FOO = 1; // comment

    protected $bar;
}
```

```yaml
# rector.yaml
parameters:
    sets:
        - type-declaration
    autoload_paths:
        - 'app/Test.php'
```

## Expected Behaviour

<!-- Was the change done incorrectly? Or should Rector skip it? -->

No errors.

--- a/rules/type-declaration/src/Rector/Property/PropertyTypeDeclarationRector.php
+++ b/rules/type-declaration/src/Rector/Property/PropertyTypeDeclarationRector.php
@@ -53,9 +53,9 @@ final class PropertyTypeDeclarationRector extends AbstractRector
         }
 
         // is already set
-        /** @var PhpDocInfo $phpDocInfo */
+        /** @var PhpDocInfo|null $phpDocInfo */
         $phpDocInfo = $node->getAttribute(AttributeKey::PHP_DOC_INFO);
-        if (! $phpDocInfo->getVarType() instanceof MixedType) {
+        if ($phpDocInfo !== null && ! $phpDocInfo->getVarType() instanceof MixedType) {
             return null;
         }
 

--- a/rules/type-declaration/src/TypeInferer/PropertyTypeInferer/VarDocPropertyTypeInferer.php
+++ b/rules/type-declaration/src/TypeInferer/PropertyTypeInferer/VarDocPropertyTypeInferer.php
@@ -5,6 +5,7 @@ declare(strict_types=1);
 namespace Rector\TypeDeclaration\TypeInferer\PropertyTypeInferer;
 
 use PhpParser\Node\Stmt\Property;
+use PHPStan\Type\MixedType;
 use PHPStan\Type\Type;
 use Rector\BetterPhpDocParser\PhpDocInfo\PhpDocInfo;
 use Rector\NodeTypeResolver\Node\AttributeKey;
@@ -14,8 +15,11 @@ final class VarDocPropertyTypeInferer implements PropertyTypeInfererInterface
 {
     public function inferProperty(Property $property): Type
     {
-        /** @var PhpDocInfo $phpDocInfo */
+        /** @var PhpDocInfo|null $phpDocInfo */
         $phpDocInfo = $property->getAttribute(AttributeKey::PHP_DOC_INFO);
+        if ($phpDocInfo === null) {
+            return new MixedType();
+        }
 
         return $phpDocInfo->getVarType();
     }

